modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/java/JavaErrorParsingTestCase.java

```diff
@@ -124,13 +129,30 @@ public void operationThrowsOverridesExtensionThrows() {
   }
 
   @Test
-  public void operationInheritsExtensionErrorThrows() {
+  public void operationAndConstructInheritExtensionErrorThrows() {
     extensionModel = loadExtension(HeisenbergWithExtensionThrows.class);
 
     OperationModel someOperation = extensionModel.getOperationModel("someOperation").get();
     Optional<ErrorModel> operationError = someOperation.getErrorModels().stream()
```

Comment:
this filtering and assertion is repeatead at least 3 times. Extract method

Reply:
Done.